iphone/Classes/KrollBridge.m

```diff
@@ -770,7 +770,7 @@ -(id)loadCommonJSModule:(NSString*)code withSourceURL:(NSURL *)sourceURL
 	 *      "/Resources/views/login/window.js"  __dirname = "views/login"
 	 */
 
-	NSString *filename = [sourceURL lastPathComponent];
+	NSString *filename = [sourceURL lastPathComponent]; // FIXME This should be the absolute path to the file!
```

Comment:
And in this case, it's some variation of the sourceURL without the preceding scheme and maybe some of the first path segments we don't "expose" (like maybe a "Resources/" prefix)?